src/OSPSuite.UI/Binders/ValueOriginBinder.cs

```diff
@@ -121,7 +120,7 @@ private void updateValueOrigin(bool canceled)
          _gridView.RowCellStyle += updateRowCellStyle;
          _gridView.CustomDrawCell += customDrawCell;
 
-         _valueOriginColumn = _gridViewBinder.Bind(x => x.ValueOrigin)
+         _valueOriginColumn = _gridViewBinder.AutoBind(x => x.ValueOrigin)
```

Comment:
Hoping that this might fix the refresh problem seen [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/643)